sass-lint.yml

```diff
@@ -26,6 +26,24 @@ rules:
   # Hound default is WARN.
   mixins-before-declarations: 0
 
+
+  # The following vendor prefixes are legitimate:
+  #
+  # -ms-grid-column
+  # -ms-grid-columns
+  # -ms-grid-column-span
+  # -ms-grid-row
+  # -ms-grid-rows
+  # -ms-grid-row-span
+  #
+  # Unforuntately sass-linter has no whitelist.
+  # However, we can allow every -ms-* property.
+  no-vendor-prefixes:
+  - 2
+  -
+    excluded-identifiers:
+      - ms
```

Comment:
Fattar inte alls den här linter yaml syntaxen. Men om den gör vad som sägs...Fanns det inte något sätt att slippa använda vendor-prefixes explicit och behöver vi i så fall ha denna i style guide?

Reply:
> Fattar inte alls den här linter yaml syntaxen.

```yaml
linter-name:
  # here comes an array
  # the first entry is to toggle enable/disable
  - 0 (off) 1 (warn) 2 (error)
  # the other entry is an options hash
  - 
    key: value
    key:
      - nestedvalue1
      - nestedvalue2
``` 

> Fanns det inte något sätt att slippa använda vendor-prefixes explicit

Nopes, there is no whitelist. You can only whitelist **prefixes** such as ms, moz and webkit.

Reply:
Hehe tänkte också att yaml syntaxen var konstigt. Men det verkar fungera lokalt så 🤷‍♂️

Reply:
Nej jag menade att jag trodde att du tidigare pratat om att man kunde stödja grid för moderna browser + IE genom något trick så att man slapp att koda med vendor prefixes hela tiden. Är nu tanken att vi tillåter detta och löser cross browser compabilitet varje gång vi använder css grid?